contracts/src/interfaces/IGateway.sol

```diff
@@ -107,5 +107,11 @@ interface IGateway {
     ) external payable;
 
     /// @dev Call transact in destinationChain
-    function transact(ParaID destinationChain, TransactMessage calldata message) external payable;
+    function transact(
+        ParaID destinationChain,
+        OriginKind originKind,
+        uint128 fee,
```

Comment:
Lets rename this to `destinationFee`, to be similar to the `destinationFee` in `sendToken`.

Reply:
https://github.com/Snowfork/snowbridge/pull/1141/commits/43b5442a488392c3557abad40558cf1dae8b8396